titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/ContainerResources.java

```diff
@@ -83,10 +83,10 @@ public ContainerResources(double cpu,
         this.memoryMB = memoryMB;
         this.diskMB = diskMB;
         this.networkMbps = networkMbps;
-        this.efsMounts = CollectionsExt.nullableImmutableCopyOf(efsMounts);
+        this.efsMounts = CollectionsExt.nullableImmutableCopyOf(nonNull(efsMounts));
```

Comment:
minor nit: these can now directly be a `ImmutableList.copyOf()`, or maybe even better, a `Collections.unmodifiableList()`

Reply:
I updated CollectionsExt to provide a method that does what you're suggesting. Hopefully that helps.